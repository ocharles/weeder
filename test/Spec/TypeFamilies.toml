roots = [ "Spec.TypeFamilies.TypeFamilyUsage.root" ]

type-class-roots = false

unused-types = true
