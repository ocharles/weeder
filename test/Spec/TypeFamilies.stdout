test/Spec/TypeFamilies/TypeFamilies.hs:9: (Type Family Instance) :: Family Bool
